app/code/Magento/Directory/Model/PriceCurrency.php

```diff
@@ -148,13 +146,10 @@ protected function getStore($scope = null)
     }
 
     /**
-     * Round price
-     *
-     * @param float $price
-     * @return float
+     * {@inheritdoc}
      */
-    public function round($price)
+    public function round($price, $precision = self::DEFAULT_PRECISION)
```

Comment:
Such change is not backward compatible as well, by the way:

> Adding parameters in public methods 
> Deprecate the method and add a new method with the new parameter(s) instead of adding them to a public method.